docker/scout/scripts/main.py

```diff
@@ -398,6 +452,21 @@ def main():
         documentation="Badgertree reward timestamp",
         labelnames=["lastCycleUnixtime"],
     )
+    #  --- ibBTC related ---
+    peak_gauge = Gauge(
```

Comment:
I think this gauge is redundant.   It seems like you would be able to get the same data out of here as you would from the composition guage by just specifying the correct params in the ibBTC_composistion guage or did I miss something.

Reply:
Renamed these gauges from @petrovska-petro so that it's a little clearer what each is for:
-  `peak_gauge` loads the Peak interface for each Peak and reports the portfolio value.
- The composition gauge loads the Sett interface for each of the underlying Setts within a Peak and reports the balance and price.

You could get the total value by summing the underlying Sett's, I guess, but this makes it a little more convenient and the interfaces/labels between the two are different, so I don't think it's too redundant.